### PR TITLE
umbilical object loading

### PR DESCRIPTION
**Pull request by X0-11**
adds the ability to load objects onto umbilicals

### CODE REVIEW

File: code/modules/halo/overmap/umbilicals.dm [hunk 1]
```diff
@@ -87,14 +87,24 @@
...
+/obj/docking_umbilical/proc/translate_obj(var/obj/to_move)
 	if(!isturf(current_connected.loc))
 		return
-	user.forceMove(current_connected.loc)
+	to_move.forceMove(current_connected.loc)
+
+/obj/docking_umbilical/MouseDrop(var/obj/over_object)
+	var/mob/user = usr
+	if(!istype(over_object)) return
+	if(istype(over_object,/obj/vehicles)) return //Yeah no vehicles-through-umbilical thanks. use a dropship.
```
Comment (DoctorOctopusTheOctopi) [inline code/modules/halo/overmap/umbilicals.dm:100]: People need to be able to put vehicles through umbilicals. It only makes sense. 🐙

**Review by DoctorOctopusTheOctopi — APPROVED** [review]
Everything looks good! 👍

File: code/modules/halo/overmap/umbilicals.dm (hunk 1 above)
Comment (X0-11) [reply code/modules/halo/overmap/umbilicals.dm:100]: but can you put umbilicals in vehicles :thinking:

Comment (DoctorOctopusTheOctopi) [reply code/modules/halo/overmap/umbilicals.dm:100]: this is what we need in our life. i want to abduct an umbilical and then drop it randomly on the battlefield, confusing everyone around me.

**Pull request merged**